crdppf/static/js/Crdppf/legalDocuments.js

```diff
@@ -126,81 +145,83 @@ Crdppf.legalDocuments.createView = function(labels) {
+                                '<p class="docurl"><b>URL:</b> <a href="#" onClick="window.open(\'{remoteurl}\');" target="_blank">{remoteurl}</a></p>',
+                            '</div>',
+                        '</tpl>',
                     '</tpl>',
                 '</tpl>',
 
                 '<h2 style="margin-top:10px;margin-bottom:5px;">Niveau cantonal</h2>',
                 '<tpl for=".">',
-                    '<tpl if="this.isLegalprovision(doctype) &amp;&amp; this.isCantonal(canton, commune)">',
-                        '<div style="font-size:10pt;padding:5px 15px;background-color:{[xindex % 2 === 0 ? "#FFF" : "#EEE"]}">',
-                            '<h3 class="doctitle"><a href="#" onClick="window.open(\'{documenturl}\');" target="_blank">{officialnb}</a> - {officialtitle}<span style="padding-left:15px;">Date de publication:</b> {publishedsince:date("d.m.Y")}<span></h3>',
-                            '<p class="docurl">URL:</b> <a href="#" onClick="window.open(\'{documenturl}\');" target="_blank">{documenturl}</a></p>',
-                            '<br />',
-                        '</div>',
+                    '<tpl if="this.isLegalprovision(doctype) &amp;&amp; this.isCantonal(state, municipalityname)">',
+                        '<tpl for=".">',
+                            '<div style="font-size:10pt;padding:5px 15px;background-color:{[xindex % 2 === 0 ? "#FFFFFF" : "#F5F5F5"]}"">',
+                                '<h3 class="doctitle"><a href="#" onClick="window.open(\'{remoteurl}\');" target="_blank">{officialnb}</a> - {officialtitle} du {sanctiondate:date("d.m.Y")}</h3>',
+                                '<p class="docurl"><b>URL:</b> <a href="#" onClick="window.open(\'{remoteurl}\');" target="_blank">{remoteurl}</a></p>',
+                            '</div>',
+                        '</tpl>',
                     '</tpl>',
                 '</tpl>',
-    
-                '<h2 style="margin-top:10px;margin-bottom:5px;">Niveau communal</h2>',
+        
                 '<tpl for=".">',
-                    '<tpl if="this.isLegalprovision(doctype)">',
-                        '<div style="font-size:10pt;padding:5px 15px;background-color:{[xindex % 2 === 0 ? "#FFF" : "#EEE"]}">',
-                            '<h3 class="doctitle"><a href="#" onClick="window.open(\'{documenturl}\');" target="_blank">{officialnb}</a> - {officialtitle}<span style="padding-left:15px;">Date de publication:</b> {publishedsince:date("d.m.Y")}<span></h3>',
-                            '<p class="docurl">URL:</b> <a href="#" onClick="window.open(\'{documenturl}\');" target="_blank">{documenturl}</a></p>',
-                            '<br />',
-                        '</div>',
+                    '<tpl if="this.isLegalprovision(doctype) &amp;&amp; this.isCommunal(state, municipalityname)">',
+                        '<h2 style="margin-top:10px;margin-bottom:5px;">Niveau communal</h2>',
```

Comment:
There is some French in here... I don't know if you want to create a multi-language app. If yes, you should do something about it...